coop_shift/model/shift_mail_registration.py

```diff
@@ -35,20 +35,24 @@ class ShiftMailRegistration(models.Model):
         'shift.registration', 'Attendee', required=True, ondelete='cascade')
     mail_ignored = fields.Boolean('Ignored', default=False)
 
-    @api.one
+    @api.multi
     def execute(self):
-        today = datetime.strftime(fields.datetime.now(),
-                                  tools.DEFAULT_SERVER_DATETIME_FORMAT)
-        # send email for user if shift hasn't started
-        if self.registration_id.shift_id.date_begin >= today:
-            # when users is in vacation, no email is sent to them.
-            if self.registration_id.partner_id.working_state in ['exempted',
-                                                                 'vacation']:
-                return
-            if self.registration_id.shift_id.is_on_holiday:
-                return
-            return super(ShiftMailRegistration, self).execute()
-        else:
-            # other case which not sending mail, marked ignore=True
-            # they won't be sent ever.
-            return self.write({'mail_ignored': True})
+        for rec in self:
+            today = datetime.strftime(fields.datetime.now(),
+                                      tools.DEFAULT_SERVER_DATETIME_FORMAT)
+            today = datetime.strptime(today,
+                                      tools.DEFAULT_SERVER_DATETIME_FORMAT)
```

Comment:
Hmmm... Are we converting a date to string, and then back to date?
Why not just use `fields.datetime.now()` ? Am I missing something?